anomdetect/src/main/java/com/expedia/adaptivealerting/anomdetect/util/MetricUtil.java

```diff
@@ -35,6 +36,8 @@
 @UtilityClass
 public class MetricUtil {
 
+    public static final String METRIC_FUNCTION_KEY = "graphite.functionTagKey";
```

Comment:
The concept of `"function"` tag is still too specific.  Can we use the term "data retrieval"?
I.e. rename to `DATA_RETRIEVAL_TAG_KEY` and `graphite.dataRetrievalTagKey`.
And make similar changes throughout all occurences.